pages/docs/getting-started/clients/python.html

```diff
@@ -0,0 +1,23 @@
+title: Connecting to Crate with Python
+author: Chris Ward
+
+## Install
+
+Use [pip](https://pypi.python.org/pypi/pip) to install the Crate Python package:
+
+```bash
+pip install crate
+```
+
+Import the client module from the `crate` package:
+
+```python
+from crate import client
+```
+## Connect to Crate
+
+Use the `client` module to create a connection:
+
+```python
+connection = client.connect(SERVER_IP)
```

Comment:
Yes, @chaudum ChrisC didn't want any queries in these opening docs, but I think I might add the cursor line at the very least as that is kind of still 'making a connection'.